core/src/main/scala/org/apache/spark/scheduler/CoarseCookSchedulerBackend.scala

```diff
@@ -106,6 +117,7 @@ class CoarseCookSchedulerBackend(
   var totalFailures = 0
   val jobIds = mutable.Set[UUID]()
   val abortedJobIds = mutable.Set[UUID]()
+  private val taskOrExecutorIdToJobId = mutable.HashMap[String, UUID]()
```

Comment:
why is this taskOrExecutorId? Isn't it just an executorId? newMesosTaskId, is just an autoincremented integer that we use for executorId which is internal to Spark. At the end of the day this is just an executorId I believe. (Correct me if I'm wrong here)